submission-template/pact22/latex/pact22-template.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[sigconf]{acmart}
+\documentclass[sigconf,anonymous=true]{acmart}
```

Comment:
We are using blind reviewing, correct?

Reply:
Yes, blind review. For the LaTeX template, looks like the following line is commonly used in submissions:
\documentclass[sigconf,review,anonymous]{acmart}

Reply:
Done: f7e0c13